src/generators/java/presets/CommonPreset.ts

```diff
@@ -110,16 +179,28 @@ export const JAVA_COMMON_PRESET: JavaPreset = {
       const shouldContainEqual = options.equal === undefined || options.equal === true;
       const shouldContainHashCode = options.hashCode === undefined || options.hashCode === true;
       const shouldContainToString = options.classToString === undefined || options.classToString === true;
+      const shouldContainMarshal = options.marshal === undefined || options.marshal === true;
```

Comment:
Since marshaling is not as common as overwriting Equal and hashcode methods I suggest we only use the following:

```suggestion
      const shouldContainMarshal = options.marshal === true;
```

This will also automatically fix the blackbox tests 🙂